icons/asciidoctor.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Asciidoctor icon</title><path d="M23 24a.8.8 0 01-1-.46L13.53 2.89 9.75 12h2.48a.8.8 0 01.78.82.79.79 0 01-.81.78H3a.79.79 0 01-.78-.81A.8.8 0 013 12h5L12.8.49a.8.8 0 01.72-.49.78.78 0 01.75.5l9.19 22.38A.81.81 0 0123 24zm-12.46-8.23H1.29a.81.81 0 00-.81.79.79.79 0 00.78.81h4.53L3.5 22.88a.79.79 0 00.42 1.05.81.81 0 001-.42l2.54-6.1h3a.8.8 0 00.82-.78.82.82 0 00-.74-.86z"/></svg>
```

Comment:
I would personally expect the logo to be [the one in the rounded square](https://github.com/asciidoctor/brand/blob/master/logo/logo-fill-bw.svg), why did you decide to omit that from the logo?

Reply:
Well they provide two versions, right?
`outline` and `fill`
The 'Fill' version however has two colors and therefore isn't suited for this project here. You could of course "stencil" it out. But that's just dumb. Nobody would stencil the important part out. A rounded square adds no value to a logo.

To give an analogy: The MacDonalds logo sits in a red square – However nobody would actually reproduce this logo in monochrome where the square is black and the famous "M" is transparent.

Or to link to the new Sublime Text logo: You would want to have the "S", not the square the _actual logo_ sits in. :)

Reply:
I agree that this is somewhat subjective. However:

- There are other examples where the logo does include a square, e.g. [IntelliJ IDEA](https://simpleicons.org/?q=intellij%20idea), [WebStorm](https://simpleicons.org/?q=webstorm), [Kentico](https://simpleicons.org/?q=kentico), and to some extend even [Adobe](https://simpleicons.org/?q=adobe).
- Asciidocter consistently uses it **with** the rounded square. Look at [the favicon of their website](https://asciidoctor.org/), their [GitHub profile](https://github.com/asciidoctor), or their [Gitter Chat](https://gitter.im/asciidoctor/asciidoctor). The only place where it is not used(/shown) is their [Twitter](https://twitter.com/asciidoctor), but I would say that is just because it is inside a circle already.
- I'm not sure what you're getting at with McDonalds. If I [search Google or MacDonalds](https://www.google.com/search?q=mcdonalds) I get the logo as Just the "M". If I got to [mcdonalds.com](https://www.mcdonalds.com/us/en-us.html) I see just the "M". I suppose it is sometimes placed in a squircle?
- Sublime is more subjective because the official website uses both versions, but it seems the version without the square is used more often.

Anyway, what do other @simple-icons/maintainers think?

Reply:
Reading the [Asciidoctor Brand & Visual Identity] (https://github.com/asciidoctor/brand/blob/master/README.adoc) README on GitHub it is clear from their specifications that:

- "The logo is square."
- "The main logo is the fill variant."
- "The round edges version is preferred"


They also (helpfully) provide a version (https://github.com/asciidoctor/brand/blob/master/logo/logo-fill-bw.svg) most suitable for use within this project. Resize...optimize... 🎉 😄

Reply:
**tldr;** While I can't agree I will still change it.

---

Your commitment to stick to official brand guides and what not is amazing, I fully respect that. I'm just arguing in favor of the logo.
While it's certainly right, that at most places the Asciidoctor logo comes in their rounded square flavor they also fail to actually give a good comparison. At no place they are in need of a monochrome version with an unknown and potentially changing background.
The version you have linked, @phatbhoy67, is also not monochrome/stenciled. (The "A" is white.)
So if you argue: Don't alter the logo! – It doesn't really work, we alter it nonetheless.

I am a trained graphic designer and personally would never do what I am doing right now :D 
It gives the rounded square (which again, adds nothing intrinsic to a logo<sup>[0]</sup>) to much visual weight and importance – while the most important part is stenciled out.

I hope you now can at least comprehend my view and why I think this decision is bad – yet being correct.

[0] Indeed I would have done it different in many of the examples you linked :)